app-backend/api/src/main/scala/scene/package.scala

```diff
@@ -1,16 +1,16 @@
 package com.azavea.rf.api
 
 import com.azavea.rf.datamodel._
+
+import io.circe._
+import cats.syntax.either._
```

Comment:
Are all of these new imports necessary? I guess to make them available for the rest of the package?